### ISSUE TITLE
Missing indentation when pressing enter after a '{'

### ISSUE THREAD

**Issue by jeremyfa** [TiVo/intellij-haxe#119]
Hi, first of all, thanks for this great plugin. It is so far the best way to work with Haxe on Mac !

I have however a problem with indentation. It may seem like a minor problem, but can be pretty annoying with time:

When pressing enter after writing some condition, function, while loop, with an opening brace, I would expect to have an additional indentation (like any other intellij plugin, or even most of the code editors). But instead, the next line is not indented by one more tab, and each time I have to type de tab myself.

Some illustation:

Expected when pressing enter to move to the next line:

```haxe
if (someCondition) {
    | // <- text cursor indented

```

Current behavior:

```haxe
if (someCondition) {
| // <- text cursor not indented

```

I don't know if this change is difficult to achieve, but fixing this problem will, in my opinion, really improve the coding experience.

I am using the latest plugin build from the ClassHierarchy branch on Mac OS X (I also experience the issue on the Intellij repository's plugin).

**Comment by EBatTiVo** [comment]
Verified using IDEA13 on ClassHierarchy branch.

**Comment by jeremyfa** [comment]
I didn't tested because I don't have the environment to rebuild the plugin, but I still did a pull request: https://github.com/TiVo/intellij-haxe/pull/120

**Comment by EBatTiVo** [comment]
Fixed.  Merged two pull requests (#120 #126).  I think this is fixed on the ClassHierarchy branch.  Please re-open if you disagree.
